# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/mdnsresponder/hotfix-shared.patch
# Copyright (C) 2008 - 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- mDNSResponder-107.6/mDNSPosix/Makefile.orig	2008-09-06 17:18:32.000000000 +1000
+++ mDNSResponder-107.6/mDNSPosix/Makefile	2008-09-06 17:19:54.000000000 +1000
@@ -574,8 +574,8 @@
 	chmod 444 $@
 	# Check the nsswitch.conf file.
 	# If 'mdns' does not already appear on the "hosts:" line, then add it right before 'dns'
-	cp -f /etc/nsswitch.conf /etc/nsswitch.conf.pre-mdns
-	sed -e '/mdns/!s/^\(hosts:.*\)dns\(.*\)/\1mdns dns\2/' /etc/nsswitch.conf.pre-mdns > /etc/nsswitch.conf
+	#cp -f /etc/nsswitch.conf /etc/nsswitch.conf.mdns
+	sed -e '/mdns/!s/^\(hosts:.*\)dns\(.*\)/\1mdns dns\2/' /etc/nsswitch.conf > /etc/nsswitch.conf.mdns
 
 #############################################################################
 
